src/chrome/forms/TextInput.tsx

```diff
@@ -48,15 +52,18 @@ const TextInput: React.FC<TextInputProps> = ({
   }
 
   return (
-    <div className={`mb-2 ${className}`}>
-      <div className="mt-1 mb-2 relative rounded-md shadow-sm">
+    <div className={`${className}`}>
+      <div className="mt-2 mb-2 relative rounded-md shadow-sm">
         {label && <InputLabel
           label={label}
           tooltip={labelTooltip}
           tooltipFor={name}
         />}
         <input
-          className="focus:ring-transparent focus:border-qripink-600 block w-full px-2 text-sm border-qrigray-300 rounded-md placeholder-qrigray-400 rounded-lg"
+          className={`focus:ring-transparent focus:border-qripink-600 block w-full px-2 text-sm border-qrigray-300 rounded-md placeholder-qrigray-400 rounded-lg ${inputClassName}`}
+          style={{
+            height: (size === 'sm') ? 24 : ''
```

Comment:
I did this change, I needed to add  `sm` for a shorter text input, and implied that the default input would be `md`.  `lg` doesn't exist but I left it in there in case we need it.  Can take it out.